tutorials/onchain-governance/fast-track-gov-proposal.md

```diff
@@ -0,0 +1,224 @@
+
+5. Open the `tsconfig.json` file and ensure it includes these compiler options:
+    ```json
+    --8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/tsconfig.json'
+    ```
+
+## Submitting and Executing a Proposal Using Chopsticks
+
+You should identify the right track and origin for your proposal. For example, select the appropriate treasury track based on the spending limits if you're requesting funds from the treasury. For more detailed information, refer to [Polkadot OpenGov Origins](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins){target=\_blank}.
+
+!!!note
+    This tutorial will focus on the main steps and core logic within the main function. For clarity and conciseness, the implementation details of individual functions will be available in expandable tabs below each section. You'll find the complete code for reference at the end of the tutorial.
+
+### Spin Up the Polkadot Fork
+
+To set up your Polkadot fork using Chopsticks, open a new terminal window and run the following command:
+
+```bash
+npx @acala-network/chopsticks --config=polkadot
+```
+
+This command will start a local fork of the Polkadot network accessible at `ws://localhost:8000`. Keep this terminal window open and running throughout your testing process.
+
+Once your forked network is up and running, you can proceed with the following steps.
+
+### Set Up Dependencies and Structure
+
+Begin by adding the necessary imports and a basic structure to the `test-proposal.ts` file:
+
+```typescript
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:imports'
+
+const main = async () => {
+  // The code will be added here
+
+  process.exit(0);
+}
+
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:try-catch-block'
+```
+
+This structure provides the foundation for your script. It imports all the necessary dependencies and sets up a main function that will contain the core logic of your proposal submission process.
+
+### Connect to the Forked Chain
+
+Create a `connectToFork` function outside the `main` function to connect your locally forked chain to the Polkadot.js API:
+
+```typescript
+    --8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:connectToFork'
+```
+
+Inside the `main` function, add the code to establish a connection to your local Polkadot fork:
+
+```typescript hl_lines="2-3"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:226:229'
+  ...
+}
+```
+
+### Create and Submit the Proposal
+
+Create a `generateProposal` function responsible for preparing and submitting the on-chain proposal. The function implements a workflow that involves the following:
+
+1. Configuring a keyring for the Alice account to facilitate transaction submission. When using Chopsticks, this development account is pre-funded to execute all necessary actions
```

Comment:
I think it's important in tutorials that we use actionable steps. These are sort of describing what the user should be doing rather than instructing the user to configure a keyring, etc.

I mean if we really need this for launch then we can address this later, but I think this is super important